src/infra/main/template.yaml

```diff
@@ -77,23 +77,23 @@ Mappings:
     staging:
       LambdaLogLevel: INFO
       AccountDeletionProcessorSNSTopicARN: "PLACEHOLDER VALUE" #TBC
-      OrchestrationVPCID: vpc-09cfe19dc5f8fcb47 # ID Reuse Core Staging VPC ID
-      AuthenticationVPCID: vpc-09cfe19dc5f8fcb47 # ID Reuse Core Staging VPC ID
-      OneLoginHomeVPCID: vpc-09cfe19dc5f8fcb47 # ID Reuse Core Staging VPC ID
+      OrchestrationVPCID: vpc-02e4eb3b0cffc08b0 # Performance testing VPC ID
```

Comment:
For staging env, could we add the VPC IDs for Auth and Orch to allow them to test against staging as well as leaving perf in the olh placeholder